PSScriptAnalyzerSettings.psd1

```diff
@@ -0,0 +1,6 @@
+# PSScriptAnalyzerSettings.psd1
+@{
+    Severity     = @('Error', 'Warning', 'Information')
+    ExcludeRules = @('PSAvoidUsingCmdletAliases' ,'PSAvoidUsingPlainTextForPassword','PSAvoidUsingWriteHost','PSAvoidUsingInvokeExpression','PSUseShouldProcessForStateChangingFunctions',
+    'PSAvoidUsingUsernameAndPasswordParams','PSUseProcessBlockForPipelineCommand','PSAvoidUsingConvertToSecureStringWithPlainText')
+}
```

Comment:
Could we add `PSUseSingularNouns` to the excludes list, as there are valid uses of plural functions :)

Reply:
Sure, but I'm going to open an improvement request to rename all internal function to singular name :)